packages/common-utils/tsconfig.base.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
```

Comment:
Is it possible for us to share the tsconfig base across api and app using this config, or are they fairly diverged?